packages/generator/src/action-function-import/operation.ts

```diff
@@ -0,0 +1,77 @@
+import { FunctionDeclarationStructure, StructureKind } from 'ts-morph';
+import voca from 'voca';
+import { VdmOperation, VdmServiceMetadata } from '../vdm-types';
+import { isEntityNotDeserializable } from '../edmx-to-vdm/common';
+import { getRequestBuilderArgumentsBase } from './request-builder-arguments';
+import { operationReturnType } from './return-type';
+
+const parameterName = 'parameters';
+
+/**
+ * @internal
```

Comment:
[pp] Add some doc to explain what this object represents: I guess function and action.